build/templates/errors.py.mako

```diff
@@ -42,12 +44,12 @@ class Error(_ErrorBase):
         super(Error, self).__init__(error_code, error_description)
 
 
-class Warning(_ErrorBase):
+class ${module_name_class}Warning(Warning):
```

Comment:
Shame to rename class, since it's already scoped inside the nidmm (or whatever) module. It's inconsistent with Error and redundant. I suppose there was no way around it that you could find?

Reply:
Not that I could find.